telegram-bot/main.py

```diff
@@ -29,14 +29,14 @@ async def main():
     dp.include_routers(router)
 
     report_callback_provider = ReportCallbackProvider(
-        path_to_schemas='../common/generated/failures-schemas.json'
+        path_to_schemas='/app/common/generated/failures-schemas.json'
```

Comment:
Hardcoded, but in Docker container this workdir is called /app

Reply:
попробуй сделать относительный путь

Reply:
бот ни про какой app не знает

Reply:
I see the problem. Will try to fix it.

You can help and change path too...

Reply:
The issue was that Docker container did not recognize ../ and I was forced to change it 😄

Reply:
This issue was connected to the **WORKDIR** in the Dockerfile

When you type CMD command, it will execute following bash script inside the folder WORKDIR. When you start the python script, you start it from the folder: `/app`
And when the code see path '../', it go upper than the `/app` folder is placed. Now I fixed it, check